BUCK

```diff
@@ -0,0 +1,22 @@
+genrule(
```

Comment:
please move this and everything relating to debian packaging into `scripts/packages/debian`